iree/tools/CMakeLists.txt

```diff
@@ -459,12 +459,11 @@ if(${IREE_BUILD_COMPILER})
     HOSTONLY
   )
 
-  if(IREE_BUILD_TESTS)
-    # Ensure FileCheck gets built. Tests don't have dependencies in CMake
-    # because they aren't targets. So until we fix that, we just force this to
-    # get built.
-    set_target_properties(FileCheck PROPERTIES EXCLUDE_FROM_ALL OFF)
-  endif()
+  # Ensure FileCheck gets built. Tests don't have dependencies in CMake because
+  # they aren't targets. So until we fix that, we just force this to get built.
+  # Limiting this to when IREE_BUILD_TESTS is set prevents the installation
+  # below, which we use for cross-platform testing.
+  set_target_properties(FileCheck PROPERTIES EXCLUDE_FROM_ALL OFF)
```

Comment:
Can you trigger https://buildkite.com/iree/iree-build-configurations on this PR? I want to verify that https://github.com/google/iree/blob/main/build_tools/cmake/build_runtime.sh doesn't build LLVM code

Reply:
Yes, automerge is doing that for you by merging this :-P

Reply:
Ah, automerge.. well, that will run on postsubmit anyways :P I can keep an eye on https://buildkite.com/iree/iree-build-configurations/builds/1887


aaand as I'm typing this it already finished. Looks good.